alonet/raft/raft.py

```diff
@@ -37,20 +37,23 @@ class RAFTBase(nn.Module):
     corr_levels = None
     corr_radius = None
 
+    out_plane = None
```

Comment:
Can we add a comment to say that corr_levels, corr_radius and out_plane must be set by the child class ?